pkg/controllers/cassandra/util/util.go

```diff
@@ -0,0 +1,37 @@
+package util
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/jetstack-experimental/navigator/pkg/apis/navigator"
+	v1alpha1 "github.com/jetstack-experimental/navigator/pkg/apis/navigator/v1alpha1"
+)
+
+const (
+	typeName = "cass"
+	kindName = "CassandraCluster"
```

Comment:
We really should come up with a better way to derive `kindName`... there's a few ways we can do it. But this isn't something to worry about now.